src/components/ProgressBarList/ProgressBarList.js

```diff
@@ -7,9 +7,7 @@ const ProgressBarList = ({
   return (
     <div className='barlist'>
             {data?.map((item, index) => (
-                <ul key={`${item.completed}_${index}`}>
-                    <li key={ index }><ProgressBar item={ item } /></li>
-                </ul>
+                <li key={index}><ProgressBar props={item} /></li>
```

Comment:
Try not to use index for the key. 
Try to use some unique value.
If you use index, it will be 0, 1, 2. Other parts in the same page will have the same key.